.changeset/wild-pandas-destroy.md

```diff
@@ -0,0 +1,8 @@
+---
+"wrangler": minor
+---
+
+feature: add support for Queue Consumer concurrency
+
+Consumer concurrency allows a consumer Worker processing messages from a queue to automatically scale out horizontally in order to keep up with the rate that messages are being written to a queue.
+The new `max_concurrency` setting can be used to limit the maximum number of concurrent consumer Worker invocations.
```

Comment:
Out if curiosity, how does queue concurrency relate to queue batches?
If you set the concurrency to 1, will you just receive larger batches in that single invocation?

Reply:
The size of batches should be unaffected. That's still totally controlled by the user's batch size setting :)
If you set concurrency to 1, batches are processed serially one at a time (which is how the system worked before the concurrency feature was added).